helm/flink-operator/values.yaml

```diff
@@ -18,9 +18,6 @@
 
 ---
 
-operatorNamespace:
-  name: default
```

Comment:
We should document this in the README so that it's clear how to select the namespace :) at least one example would be good

Reply:
> I think the change makes sense but could you please open a JIRA ticket first?

Sure, will do~

Reply:
https://issues.apache.org/jira/browse/FLINK-26594

Reply:
perfect, thank you :)